### ISSUE TITLE
Remove exception stacktraces from tests

### ISSUE THREAD

**Issue by tomas-langer** [oracle/helidon#466]
A lot of tests print stacktraces of expected exceptions - these should be updated not to pollute the log output.
Also some tests print a huge stacktrace due to HK2 missing a dependency (activation `DataSource` and/or other libraries that were part of JDK before Java 9) - this generates too much data and as a result the standard output of wercker is not really usable to analyze behavior of the build.

**Comment by tomas-langer** [comment]
Example (I think from FT TCK):
```
WARNING: HK2 service reification failed for [org.glassfish.jersey.message.internal.DataSourceProvider] with an exception:
MultiException stack 1 of 2
java.lang.NoClassDefFoundError: javax/activation/DataSource
```

**Comment by barchetta** [comment]
The "HK2 service reification" exceptions are fixed by PR #468.

**Comment by m0mus** [comment]
@tomas-langer said that I can close it.
